core/src/main/scala/filodb.core/memstore/DemandPagedChunkStore.scala

```diff
@@ -73,21 +73,23 @@ extends RawToPartitionMaker with StrictLogging {
       tsShard.shardStats.numChunksPagedIn.increment(rawPartition.chunkSets.size)
       // One chunkset at a time, load them into offheap and populate the partition
       rawPartition.chunkSets.foreach { case RawChunkSet(infoBytes, rawVectors) =>
-        val memFactory = getMemFactory(timeBucketForChunkSet(infoBytes))
-        val chunkID = ChunkSetInfo.getChunkID(infoBytes)
+        if (!rawVectors.isEmpty) {
```

Comment:
Please add a comment detailing the bug about metadata allocation error that might possibly happen that this fixes